### ISSUE TITLE
document is named "fakedoc"

### ISSUE THREAD

**Issue by garfieldnate** [desilinguist/ibleu#1]
Whatever the name of the document I load, the DocId is presented as "fakedoc".

**Comment by desilinguist** [comment]
Are you using text files or xml files? The fakedoc trick is used for text
files since it can't contain any metadata.

**Comment by garfieldnate** [comment]
Yes, I'm using text files. After debugging the application for #2 I saw how this trick worked. As a user it surprised me, though, because my text files were not named "fakedoc", and I was expecting the display to show the name of the hypothesis file (minus `.txt`).

**Comment by desilinguist** [comment]
Indeed. That would be a better option. It just didn't occur to me to do
that when I first wrote this originally. Pull request welcome :)

Thanks for the Chrome stuff too.

**Comment by desilinguist** [comment]
BTW, the fakedoc trick is fully documented in the readme so I don't think
it's that big a deal.

**Comment by garfieldnate** [comment]
Yeah, it's not a show-stopper. I just think it would make more sense to the user to use a file name or something.

**Comment by desilinguist** [comment]
I agree.

**Comment by desilinguist** [comment]
I am happy to take pull requests for this as I don't really have time to work on this for now.

**Comment by Cuzzie** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.n7n-dev, @alittlebitfun, @chc273, @deism, @RuslanJabbar, @digiaries, @zaigie, @alaypanov, @honggui, @baireslearn, @bieli, @xujiesh0510, @velocitatem, @plamengj, @Shellishack, @Adityakhurana9499, @PolinaBortnik, @ww-jermaine, @SouthernMine4957, @serpro69
